packages/firestore/test/lite/integration.test.ts

```diff
@@ -2038,3 +2045,137 @@
     });
   });
 });
+
+describe('countQuery()', () => {
+  const testDocs = [
+    { author: 'authorA', title: 'titleA' },
+    { author: 'authorA', title: 'titleB' },
+    { author: 'authorB', title: 'titleC' },
+    { author: 'authorB', title: 'titleD' },
+    { author: 'authorB', title: 'titleE' }
+  ];
+
+  it('AggregateQuery and AggregateQuerySnapshot inherits the original query', () => {
+    return withTestCollection(async coll => {
+      const query_ = query(coll);
+
+      const countQuery_ = countQuery(query_);
```

Comment:
Please remove the `expect` that checks the `type` property; this is an implementation detail that doesn't need to be tested.